foundry.lock

```diff
@@ -0,0 +1,26 @@
+{
+  "lib/account-abstraction": {
+    "rev": "abff2aca61a8f0934e533d0d352978055fddbd96"
+  },
+  "lib/forge-std": {
+    "tag": {
+      "name": "v1.10.0",
+      "rev": "8bbcf6e3f8f62f419e5429a0bd89331c85c37824"
+    }
+  },
+  "lib/openzeppelin-contracts": {
+    "rev": "5705e8208bc92cd82c7bcdfeac8dbc7377767d96"
+  },
+  "lib/p256-verifier": {
+    "rev": "29475ae300ec95d98d5c7cc34c094846f0aa2dcd"
+  },
+  "lib/safe-singleton-deployer-sol": {
+    "rev": "cf2b89c33fed536c4dd6fef2fb84f39053068868"
+  },
+  "lib/solady": {
+    "rev": "c4c96607cb3aa3807b14c81ae2015bcba061f8fc"
```

Comment:
if this is a commit, we actually want this to be `5ea5d9f57ed6d24a27d00934f4a3448def931415`